### PR TITLE
Define MIXPANEL_PROJECT_TOKEN

### PR DESCRIPTION
**Pull request by iokpala**
[Zenhub](https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/295)

Related to https://github.com/cds-snc/notification-admin/pull/1208

## What are you changing?
- [ ] Releasing a new version of Notify
- [x] Changing kubernetes configuration

## Provide some background on the changes
 Adding MIXPANEL_PROJECT_TOKEN, this token will be used to identify Notify on Mixpanel

## If you are releasing a new version of Notify, what components are you updating
- [ ] API
- [x] Admin
- [ ] Documentation
- [ ] Document download API
- [ ] Document download frontend

## Checklist if releasing new version:
- [x] I made sure that the changes are as expected in [Notify staging](https://staging.notification.cdssandbox.xyz/)
- [ ] I have checked if the docker images I am referencing exist
    - [ ] [api lambda](https://ca-central-1.console.aws.amazon.com/ecr/repositories/private/296255494825/notify/api-lambda?region=ca-central-1) (requires Notification-Production / AdministratorAccess login)
    - [ ] [api k8s](https://gallery.ecr.aws/v6b8u5o6/notify-api)
    - [x] [admin](https://gallery.ecr.aws/v6b8u5o6/notify-admin)
    - [ ] [documentation](https://gallery.ecr.aws/v6b8u5o6/notify-documentation)
    - [ ] [document download API](https://gallery.ecr.aws/v6b8u5o6/notify-document-download-api)
    - [ ] [document download frontend](https://gallery.ecr.aws/v6b8u5o6/notify-document-download-frontend)

## Checklist if making changes to Kubernetes:
- [ ] I know how to get kubectl credentials in case it catches on fire

## After merging this PR
- [ ] I have verified that the tests / deployment actions succeeded
- [ ] I have verified that any affected pods were restarted successfully
- [ ] I have verified that I can still log into [Notify production](https://notification.canada.ca)
- [ ] I have verified that the smoke tests still pass on production
- [ ] I have communicated the release in the #notify Slack channel.

### CODE REVIEW

File: base/admin-deployment.yaml [hunk 1]
```diff
@@ -68,6 +68,8 @@ spec:
               value: 'application.py'
             - name: NOTIFY_ENVIRONMENT
               value: '$(ENVIRONMENT)'
+            - name: MIXPANEL_PROJECT_TOKEN
```
[the remark this answers is not in the document]
Comment (iokpala) [reply base/admin-deployment.yaml:71]: I don't think so

[the remark this answers is not in the document]
Comment (sastels) [reply base/admin-deployment.yaml:71]: hrm, where is the value of the token coming from?

[the remark this answers is not in the document]
Comment (iokpala) [reply base/admin-deployment.yaml:71]: I have updated the `MIXPANEL_PROJECT_TOKEN` in  kustomization.yaml

**Review by sastels — APPROVED** [review]
LGTM! 🚀

File: README.md [hunk 2]
```diff
@@ -35,16 +35,26 @@ AWS_PROFILE=notify-staging make decrypt-staging
...
 You should leverage the appropriate commands in the Makefile:
 - the `make encrypt-staging` command that will encrypt environment variables in staging ;
-- the `make encrypt-production` command that will encrypt environment variables in production.
 
 ```sh
 AWS_PROFILE=notify-staging make decrypt-staging
 # Change values in the decrypted file at env/staging/.env
 # Encrypt the decrypted file that you just edited
-AWS_PROFILE=notify-staging make decrypt-staging
+AWS_PROFILE=notify-staging make encrypt-staging
```
Comment (jimleroyer) [inline README.md:43]: 😅

**Review by jimleroyer — APPROVED** [review]
:shipit: 

Thanks for the doc improvements and cleaning up.

**Pull request merged**